src/h5web/vis-packs/core/shared/Html.tsx

```diff
@@ -1,56 +1,59 @@
-import { useState, useEffect, ReactElement } from 'react';
+import { useState, useEffect, forwardRef } from 'react';
 import ReactDOM from 'react-dom';
 import { useThree } from 'react-three-fiber';
 
 // Simplified version of `drei`'s `<Html>` component
 // https://github.com/pmndrs/drei/blob/v2.2.3/src/Html.tsx
-function Html(props: React.HTMLAttributes<HTMLDivElement>): ReactElement {
-  const { className, style, children, ...divProps } = props;
+const Html = forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDivElement>>(
+  (props, ref) => {
```

Comment:
Just adding the possibility to do `<Html ref={ref} />` (by wrapping the component in `forwardRef`).